bitrise.yml

```diff
@@ -1418,6 +1418,47 @@ workflows:
       bitrise.io:
         stack: osx-xcode-14.1.x-ventura
         machine_type_id: g2.8core
+  perfherder-test:
+    steps:
+    - cache-pull@2.1:
+        is_always_run: true
+    - xcode-build-for-simulator@0.12:
+        inputs:
+        - test_plan: PerformanceTestPlan
+        - xcodebuild_options: CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO
+        - scheme: Fennec_Enterprise_XCUITests
+    - xcode-test@4.5:
+        inputs:
+        - scheme: Fennec_Enterprise_XCUITests
+        - xcodebuild_test_options: "-testPlan PerformanceTestPlan"
+        - simulator_device: "iPhone 14"
+        - test_plan: PerformanceTestPlan
+    
+    - deploy-to-bitrise-io@1.9: {}
+    - script@1.1:
+        title: Create perfherder data object
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+            # get dependency
+            cd ./test-fixtures && git clone https://github.com/isabelrios/xcresult_extract.git
+            
+            xcrun xcresulttool graph --path $BITRISE_XCODEBUILD_TEST_LOG_PATH/Test-Fennec_Enterprise_XCUITests-*
```

Comment:
If I'm not wrong, the file we need is xcresult, can you use this env variable instead? `$BITRISE_XCRESULT_PATH`

The one you are using goes to this path: `/Users/vagrant/deploy/xcodebuild_test.log` which does not contain that file.
The file structure and naming is the default generated by xcode, so apart from chaning the base directory, the rest is the same

Reply:
updated to new env var and running build to test